tracer/test/Datadog.Trace.Security.IntegrationTests/ApiSecurity/ruleset-with-block.json

```diff
@@ -0,0 +1,9323 @@
+{
```

Comment:
If we're going to use custom rule sets for the integration tests, we should consider creating small rule files with just the rules required. I've added big rule files in several places, so no need to make the change in this PR, but it's a refactoring that would make things easier;

Reply:
Fixed in [3c11abb](https://github.com/DataDog/dd-trace-dotnet/pull/4942/commits/3c11abb2880c8cd81e70c7b5846ab05dd564e5e0)